package.json

```diff
@@ -77,6 +77,6 @@
     "webpack-dev-server": "2.4.2"
   },
   "dependencies": {
-    "prop-types": "15.5.8"
+    "prop-types": "15.5.10"
```

Comment:
would be better to use `^15.5.8`, which is `15.5.10` compatible and also will not change the "base" version